CrashReporter/mainwindow.cpp

```diff
@@ -31,25 +31,27 @@ MainWindow::MainWindow(QWidget *parent, const QString& path) :
         _script = qApp->applicationDirPath() + QDir::separator() + "report_issue.js";
     #endif
 
-    #ifdef Q_OS_MAC
-        _script = qApp->applicationDirPath() + QDir::separator() + ".." + QDir::separator() + ".." \
-                + QDir::separator() + ".." + QDir::separator() + "report_issue.js";
+    #if defined(Q_OS_MAC) | defined(Q_OS_WIN)
+        connect(uploader, &FileUploader::uploadDone, this, &MainWindow::uploadFinished);
     #endif
 
+//    #ifdef Q_OS_MAC
+//        _script = qApp->applicationDirPath() + QDir::separator() + ".." + QDir::separator() + ".." \
+//                + QDir::separator() + ".." + QDir::separator() + "report_issue.js";
+//    #endif
 
-    #if defined(Q_OS_UNIX)
-        QString nodePath = QStandardPaths::findExecutable("node");
-        #ifdef Q_OS_LINUX
 
+    #ifdef Q_OS_LINUX
+        QString nodePath = QStandardPaths::findExecutable("node");
         if(!QStandardPaths::findExecutable("node", QStringList() << qApp->applicationDirPath()).isEmpty())
             nodePath = qApp->applicationDirPath() + QDir::separator() + "node";
-        #endif
 
-        #ifdef Q_OS_MAC
-        QString binDir = qApp->applicationDirPath() + QDir::separator() + ".." + QDir::separator() + ".." + QDir::separator() + ".." + QDir::separator();
-        if(!QStandardPaths::findExecutable("node", QStringList() << binDir + "node_bin" + QDir::separator() ).isEmpty())
-            nodePath = binDir + "node_bin" + QDir::separator() + "node";
-        #endif
+
+//        #ifdef Q_OS_MAC
```

Comment:
@rish9511 Can we remove all the commented out code? Assuming it's not going to be used from now on.